Sources/DrawerPresentation/DrawerInteractionDelegate.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+@MainActor
+public protocol DrawerInteractionDelegate: AnyObject, Sendable {
+    func drawerInteraction(_ interaction: DrawerInteraction, widthForDrawer drawerViewController: UIViewController) -> CGFloat
+    func drawerInteraction(_ interaction: DrawerInteraction, presentingViewControllerFor viewController: UIViewController) -> UIViewController?
+    func viewController(for interaction: DrawerInteraction) -> UIViewController
```

Comment:
API Design Ref:
[func viewController(for: UILargeContentViewerInteraction) -> UIViewController](https://developer.apple.com/documentation/uikit/uilargecontentviewerinteractiondelegate/3183929-viewcontroller)